portal-web/test/functional/com/liferay/portalweb/macros/util/File.macro

```diff
@@ -10,12 +10,23 @@ definition {
 		return ${extension};
 	}
 
-	@description = "Replace the first occurence of ${regex} in ${fileName} with ${content}"
+	@description = "Replace all occurencences of ${regex} in ${fileName} with ${content}"
```

Comment:
@Kvale1733 This is technically a breaking change. There are a few usages of this. I say keep as is and create a regexReplaceAll macro instead since we only need to add a new functionality and dont need to risk breaking other tests.